REFERENCE.md

```diff
@@ -179,12 +193,12 @@ in a hash.
 * [`swapcase`](#swapcase): This function will swap the existing case of a string.
 * [`time`](#time): This function will return the current time since epoch as an integer.
 * [`to_bytes`](#to_bytes): Converts the argument into bytes, for example 4 kB becomes 4096.
-* [`to_json`](#to_json): Convert a data structure and output to JSON
```

Comment:
@chelnak You've hit this [bug](https://github.com/puppetlabs/puppet-strings/issues/296).

Reply:
wowzer good spot @alexjfisher !